EIPS/eip-6916.md

```diff
@@ -125,7 +125,7 @@ Genesis Validators => Epochs until < 66% majority
 
 #### ChainId 
 
-ChainId is a variable because it needs to keep changing with each new genesis to avoid replay attack. The function for the new ChainId value is a simple iteration (+1). The ChainId in `genesis 0` is hardcoded constant and network increments it with each new gensis. 
+ChainId is a variable because it needs to keep changing with each new genesis to avoid replay attack. The function for the new ChainId value is a simple incrementation (+1). The ChainId in `genesis 0` is hardcoded constant and network increments it with each new gensis. 
 
 It shouldn't collide with any other existing EVM chain even after longer period of iterations. 
```

Comment:
suggested: "after many iterations"
maybe we could add reference to existing chainIds here to aid readability